pkg/cmd/rpc_server.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	server "github.com/threefoldtech/tf-grid-cli/pkg/server"
+)
+
+func RPCServer() error {
```

Comment:
Maybe rename to Server? The RPC part is kind of redundant. Maybe do so for the name of the file too.

Reply:
good idea..
i just don't want to rename it to server because there is a package called server, maybe TFGridServer???